serviceconnector/src/createServiceConnector/CreateLinkerStep.ts

```diff
@@ -0,0 +1,32 @@
+/*---------------------------------------------------------------------------------------------
+*  Copyright (c) Microsoft Corporation. All rights reserved.
+*  Licensed under the MIT License. See License.md in the project root for license information.
+*--------------------------------------------------------------------------------------------*/
+
+import { AzureWizardExecuteStep, nonNullValue } from "@microsoft/vscode-azext-utils";
+import { createServiceConnectorClient } from "../serviceConnectorClient";
+import { ICreateLinkerContext } from "./ICreateLinkerContext";
+
+export class CreateLinkerStep extends AzureWizardExecuteStep<ICreateLinkerContext>{
```

Comment:
nit: Generally, we call things `<noun>CreateStep`. Just makes it easier to organize/read (otherwise they would all start with Create.